### DISCUSSION TITLE
Thanks Gradience team for ability to theme latest GNOME

### DISCUSSION THREAD

**Discussion by Quatters** [GradienceTeam/Gradience#337] [General]
Hey everyone, just want to say that I'm appreciate the work already done by Gradience team. Since GNOME 42 and libadwaita release I had to accept that theming is dead. Before that I used cool theme with nord colors and awesome macos-like titlebuttons like on the screenshot. Actually, with Gradience not only buttons work (and work with libadwaita, gtk-4 and gtk-3 apps!), but accent and background colors too of course. Colors, however, not works with gnome-shell for now, so I'm looking forward for this feature.
Special thanks to css-injection feature developers :)

![изображение](https://user-images.githubusercontent.com/73882772/188315350-e588cb31-bd9a-49cf-90f1-d334f85af38a.png)

**Comment by 0xMRTT** [comment]
Thanks you :+1:
